pkg/common/vars.go

```diff
@@ -28,6 +28,7 @@ var (
 		"wav",
 		"flac",
 		"webp",
+		"pdf",
```

Comment:
how sure are we about this? these can presumably be opened up and inspected, right? do we get false positives from them or something?

Reply:
I've never seen a result in a PDF unless it has gone through a PDF handler (not yet released)